### PR TITLE
feat(package): yarn lerna version => pnpm changeset version

### PR DESCRIPTION
**Pull request by joinmouse**

### CODE REVIEW

File: web/playground/index.html
Comment (hyf0) [inline web/playground/index.html:1]: I don't think this change is related to this PR. Could you keep it original?

File: packages/rolldown/src/options/input-options.ts
Comment (hyf0) [inline packages/rolldown/src/options/input-options.ts:1]: I don't think this change is related to this PR. Could you keep it original?

**Review by hyf0 — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "docs": "pnpm --filter rolldown-docs run dev",
     "docs:build": "pnpm --filter rolldown-docs run build",
     "docs:preview": "pnpm --filter rolldown-docs run preview",
-    "version": "yarn lerna version --force-publish --no-private --no-git-tag-version && node scripts/post-version.cjs",
+    "version": "pnpm changeset version",
```
Comment (milesj) [inline package.json]: we should maybe remove this since we have it in the `justfile`.

File: packages/rolldown/src/options/input-options.ts
Comment (joinmouse) [reply packages/rolldown/src/options/input-options.ts:1]: I pnpm run prettier , fmt this file, now rolled back this file.

File: package.json (hunk 1 above)
Comment (joinmouse) [reply package.json]: @hyf0 What do you think？

Comment (hyf0) [reply package.json]: Yeah. I agree with @milesj. He knows this better.

Comment (joinmouse) [reply package.json]: Ok, Now I remove this script about version.

**Comment by hyf0** [comment]
Run `just fmt` to fix format errors.

**Comment by joinmouse** [comment]
> Run `just fmt` to fix format errors.

Now, fix fmt.

**Review by hyf0 — APPROVED** [review]
Thank you!

**Pull request merged**